.github/ISSUE_TEMPLATE.md

```diff
@@ -8,11 +8,18 @@
 
 [this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]
 
+**Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before.
+Please check first before submitting so that we do not have to handle and close duplicates!
```

Comment:
> I think we've generally handled usage questions on the issue tracker from time-to-time.

Yes, but that is not the purpose of the github issue tracker, and give only clutter (that said, I am very well aware it is not avoidable to a certain extent, but giving those pointers might help)